### ISSUE TITLE
Browser freezes when 'width' is not of type 'number'

### ISSUE THREAD

**Issue by automagisch** [a2800276/hexy.js#8]
When one wants to change the 'width' of a hexy output and this 'width' isn't of type number, Hexy freezes the entire browser, eventually to crash fatally. 

This is an easy fix on the client side of course using something like JS's parseInt, but I recommend implementing it into Hexy for optimized user experience. This happens in the browser version though, not tested it in the node version! 

this occured in the last stable version of Chrome, dunno about the other browsers.

Thanks! 

For people running into this, this is a quickfix:

```javascript
var w = '64'; // your width as a string, e.g from a html data-attribute

hexy(str, {
    width: parseInt(w)
});
```

**Comment by a2800276** [comment]
Interesting! Thanks, I'll have a look.

**Comment by a2800276** [comment]
Thanks again, fixed in 158e552. 
0.2.9

**Comment by a2800276** [comment]
urgh, used backtick string interpolation in new tests so tests on old node versions are failing ...
